app/assets/javascripts/discourse/app/components/modal/create-invite.gjs

```diff
@@ -331,6 +333,14 @@ export default class CreateInvite extends Component {
             @onRegisterApi={{this.registerApi}}
             as |form|
           >
+            <form.Field
+              @name="description"
+              @title={{i18n "user.invited.invite.description"}}
+              @format="large"
+              as |field|
+            >
+              <field.Input />
+            </form.Field>
```

Comment:
I think it will be useful to show a client side validation error when the value of the input exceeds the maximum length.

Reply:
Added!